cmd/hologram-server/main.go

```diff
@@ -76,6 +76,7 @@ func main() {
 		enableLDAPRoles  = flag.Bool("ldaproles", false, "Enable role support using LDAP directory.")
 		roleAttribute    = flag.String("roleattribute", "", "Group attribute to get role from.")
 		defaultRoleAttr  = flag.String("defaultroleattr", "", "User attribute to check to determine a user's default role.")
+		groupClassAttr   = flag.String("groupclassattr", "", "LDAP objectclass to determine the groups in LDAP.")
```

Comment:
You could set the default value here as `groupOfNames` and avoid the conditional on line 154.

Reply:
The reason I didn't do that is because the config is read in from the config file on line 103, and this flag should be settable from both the config file and the command line, with the command line taking precedence _if set_. If I set the default value here, I wouldn't able to tell on lines 154-158 if the value was explicitly passed in on the command line with the value of `groupOfNames` or if it was not specified and it's getting `groupOfNames` from the default value, and thus it would effectively always override anything specified in the config file with the default, which wouldn't be desirable. Is there a better way of handling this behavior?

Reply:
Ah, I see - that makes total sense. 👍